src/main/java/frc/robot/oi/XboxOperatorController.java

```diff
@@ -0,0 +1,19 @@
+package frc.robot.oi;
+
+import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+
+public class XboxOperatorController implements DriverControls, OperatorControls {
```

Comment:
If you want it to extend from both, rename to `SingleUserXboxControls`
```suggestion
public class XboxOperatorController implements OperatorControls {
```